templates/global/dashboard-nav.php

```diff
@@ -17,5 +17,7 @@
     }
 
     echo wp_kses( dokan_dashboard_nav( $active_menu ), $allowedposttags );
+
+    do_action( 'dokan_dash_sidebar' );
```

Comment:
Pls add two hooks `dokan_dashboard_sidebar_start` and  `dokan_dashboard_sidebar_end`